WordScraper/WordScraper.js

```diff
@@ -0,0 +1,113 @@
+const fs = require('fs');
+const axios = require('axios');
+const cheerio = require('cheerio');
+
+
+// Function to fetch HTML content and extract text
+const fetchTextContent = async (url) => {
+  try {
+    const response = await axios.get(url);
+    const $ = cheerio.load(response.data);
+    const textContent = $('body').text().replace(/\r/g, '').trim();
+    return textContent;
+  } catch (error) {
+    console.error(`Error fetching ${url}: ${error}`);
+    return null;
+  }
+};
+
+
+// Function to read URLs and words from files, and validate them
+function readUrlAndWords(){
+
+  let urlsSet = new Set() ;
+  let wordsSet = new Set();
```

Comment:
Please use 'const' here. Variables that are not going to be changed shouldn't be declared with 'var' or 'let'.